ansible-vault should support script for --vault-password-file too

sivel opened the pull request:
Somehow I completely missed adding support to `ansible-vault` to allow `--vault-password-file` to be a script.

This PR adds this functionality to match the now existing functionality of `ansible` and `ansible-playbook`.

sivel committed:
If ansible and ansible-playbook accept a script for --vault-password-file so should ansible-vault

mpdehaan commented:
Looks like this may need to also apply to the new "view" command on ansible-vault as well:

https://github.com/ansible/ansible/blob/devel/bin/ansible-vault

sivel commented:
I've updated this as requested. I have rebased again, and updated the view command as well.

I must have rebased last before the view command was added.

mpdehaan commented:
Merged, thank you!
